spec/requests/admin/users_manage_spec.rb

```diff
@@ -109,6 +109,8 @@ def full_profile
     end
 
     it "merges an identity on a single account into the other" do
+      omniauth_mock_twitter_payload
+      omniauth_mock_github_payload
```

Comment:
I assume we need to mock github here for the create(:identity) without a specific provider?

Reply:
Github is specified in the identity factory by default, so yes.